### PR TITLE
Update DG and UG

### PR DESCRIPTION
**Pull request by Wincenttjoi**
Changes made:
- Add numbering for all the sections
- Break devops, doc, logging, testing into individual components
- Logging and Configuration is parked under implementation
- Update help window with new feature of delivery and fix missed out metric attribute
- Added MSS and Manual Testings
- Change Quickstart in UG

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -388,22 +388,46 @@ If there is a need for any changes in the help message, `HELP_SUMMARY` can be fo
...
+*  Log messages are output through the console and to a `.log` file.
+*  The output logging level can be controlled using the `logLevel` setting in the configuration file (See the [Configuration guide](#configuration) section).
+* **When choosing a level for a log message**, follow these conventions:
+    * `SEVERE`: A critical problem detected which may cause the termination of the application.
+    * `WARNING`: Can continue, but with caution.
+    * `INFO`: Information showing the noteworthy actions by the App.
+    * `FINE`: Details that is not usually noteworthy but may be useful in debugging e.g. print the actual list instead of just its size.
+
+
+<a name="configuration></a>
```
Comment (Choo-Xing-Yu) [inline docs/DeveloperGuide.md]: is this supposed to wrap around 4.7?

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 2]
```diff
@@ -59,8 +59,8 @@ public static final String getHelpStart() {
                         + "Adding a pending delivery is very similar to this,\n"
                         + "with just a slight difference in the command. You can enter the following "
                         + "command to add a delivery:\n"
-                        + "`add-d n/NAME p/PHONE a/ADDRESS o/ORDER`\n"
-                        + "Example: `add-d n/DAMITH p/91231231 a/Jln Burong no 92 o/Mie Rebus`\n")
+                        + "`add-d n/NAME p/PHONE a/ADDRESS o/ORDER [by/TIME]`\n"
+                        + "Example: `add-d n/DAMITH p/91231231 a/Jln Burong no 92 o/Mie Rebus by/15`\n")
```
Comment (Choo-Xing-Yu) [inline src/main/java/seedu/address/commons/core/Messages.java]: misspell mee

Comment (Wincenttjoi) [reply src/main/java/seedu/address/commons/core/Messages.java]: AHHAHA SO CUTE THANKS

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Wincenttjoi) [reply docs/DeveloperGuide.md]: Thanks for spotting.
On the side note, if you put anchor tag inside ### there will be a bug. If you notice in our UG, you cant click on Features and glossary 🤪 I've changed it to outside to remove the bug

Comment (Choo-Xing-Yu) [reply docs/DeveloperGuide.md]: Ah got it! Thank you!!

File: src/main/java/seedu/address/commons/core/Messages.java (hunk 2 above)
Comment (Choo-Xing-Yu) [reply src/main/java/seedu/address/commons/core/Messages.java]: LMAO

**Review by Choo-Xing-Yu — APPROVED** [review]
LGTM

**Pull request merged**